srcpkgs/nvme-cli/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'nvme-cli'
 pkgname=nvme-cli
```

Comment:
* remove `-f` from rm commands in post_install
* if /usr/lib/systemd only contains services, we shouldn't be removing them

Reply:
> - if /usr/lib/systemd only contains services, we shouldn't be removing them

Why? :thinking:

Reply:
Ah, I see it's mentioned in `Manual.md`

> If the package installs a systemd service file or other unit, leave it in place as a
reference point so long as including it has no negative side effects.

But I still don't understand. What does it mean by _reference point_?

Reply:
(done, btw.)

Reply:
> What does it mean by _reference point_?

Frankly speaking, for "documentation purposes".

You can refer to systemd services when writing a runit service in case the target software is obscure and/or harmful to deal with.